README.md

```diff
@@ -1,2 +1,32 @@
-# SSS
-Database Code
+# Self-Sufficiency Standard Database
+
+## Background
+[The Self-Sufficiency Standard](https://selfsufficiencystandard.org/)(SSS) was created by the Center of Women's Welfare (CWW) at the University of Washington as an alterantive to the Official Poverty Measure (OPM). The Self-Sufficiency Standard data is spread across the CWW website and this repository creates a database to hold the Self-Sufficiency Standard data. 
+
+## Setting Up Computer
+It's important to set up your machine these [instructions](https://neurohackademy.org/setup/).
+
+## Cloning the Repository
```

Comment:
Let's nest all these headings about setup under the "Setting up" header by adding a `#` to make them a lower-level header.